proto/gen_attest.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-protoc -I. -I`go list -m -f "{{.Dir}}" github.com/google/go-sev-guest` --go_out=. --go_opt=module=github.com/google/go-tpm-tools/proto attest.proto
+protoc -I. -I`go list -m -f "{{.Dir}}" github.com/google/go-sev-guest` -I`go list -m -f "{{.Dir}}" github.com/google/go-tdx-guest` --go_out=. --go_opt=module=github.com/google/go-tpm-tools/proto attest.proto
```

Comment:
nit: newline EOF

Reply:
There is no new line at the EOF.
Do you want me to add a blank line?

Reply:
yes, because I think the original file has a new line at the end

Reply:
Done